### DISCUSSION TITLE
ChatGPT Integration

### DISCUSSION THREAD

**Discussion by gur26001** [refinedev/refine#4602] [Ideas]
Hi, I think this project can be made into human-command based automatic web application form which basically takes the input from the user while installing and can create/add features as per mentioned by user,using chatgpt api.
This will be something, creating website using AI(low code,no code feature).

**Comment by omeraplak** [comment]
Hey @gur26001 ,
Thank you for your suggestion! We're actually working on something similar currently. Stay tuned for updates on our progress.
